test/test_jit_string.py

```diff
@@ -278,18 +279,20 @@ def test_split():
         self.checkScript(test_split, ())
 
         def test_rsplit():
-            # type: () -> Tuple[List[str], List[str], List[str], List[str], List[str], List[str], List[str], List[str], List[str]]
             return (
                 "a a a a a".rsplit(),
-                " a a a a a ".rsplit(" "),
-                "a a a a a ".rsplit(" ", 10),
-                "a a a a a ".rsplit(" ", -1),
-                "a a a a a ".rsplit(" ", 3),
-                " a a a a a ".rsplit("*"),
-                " a*a a*a a ".rsplit("*"),
-                " a*a a*a a ".rsplit("*", -1),
-                " a*a a*a a".rsplit("a*", 10),
+                "b\nb\tb b".rsplit(),
+                "b\nb\t b b".rsplit(),
+                " c c c c c ".rsplit(" "),
+                "d d d d d ".rsplit(" ", 10),
+                "e e e e e ".rsplit(" ", -1),
+                "f f f f f ".rsplit(" ", 3),
+                " g g g g g ".rsplit("*"),
+                " h*h h*h h ".rsplit("*"),
+                " i*i i*i i ".rsplit("*", -1),
+                " j*j j*j j".rsplit("j*", 10),
             )
+
         self.checkScript(test_rsplit, ())
```

Comment:
can you also add an `if __name__ == __main__`  thing at the bottom so we can run this test in isolation?